plugins/woocommerce/tests/api-core-tests/tests/shipping/shipping-method.test.js

```diff
@@ -30,11 +35,91 @@ const methodCostIndex = 2;
  */
 test.describe('Shipping methods API tests', () => {
 
+	test('cannot create a shipping method', async ({
+		request,
+	}) => {
+		// call API to create a webhook
```

Comment:
We just need to update these comments to refer to shipping methods, as they're currently mentioning webhooks--both here and on lines 59, 73, and 86.

Reply:
Nice spot @zhongruige - thanks a lot for taking a peek!